src/openms/source/QC/TIC.cpp

```diff
@@ -54,6 +54,18 @@ namespace OpenMS
         result.retention_times.push_back(p.getRT());
       }
 
+      UInt max_int = *max_element(result.intensities.begin(), result.intensities.end());
+      for (const auto& i : result.intensities)
+      {
+        if (i == 0)
+        {
+          result.relative_intensities.push_back(0);
```

Comment:
dividing 0 by max_int should not be a problem, right?
(but maybe max_int should be checked for being non-zero?